SchedulerJobs/SchedulerJobs/Functions/AnonymiseHearingsConferencesAndDeleteAadUsersFunction.cs

```diff
@@ -49,5 +52,32 @@ await _anonymiseHearingsConferencesDataService.AnonymiseHearingsConferencesDataA
                 await _jobHistoryService.UpdateJobHistory(GetType().Name, jobSucceeded);
             }
         }
+
+        /// <summary>
+        ///     Function to anonymise hearing and conference data older than three months
+        /// </summary>
+        /// <param name="log"></param>
+        [FunctionName("AnonymiseHearingsConferencesAndDeleteAadUsersFunctionHttp")]
+        public async Task Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)]
+            HttpRequest req, ILogger log)
+        {
+            try
+            {
+                await _anonymiseHearingsConferencesDataService.AnonymiseHearingsConferencesDataAsync()
+                    .ConfigureAwait(false);
+                jobSucceeded = true;
+                log.LogInformation(LogInformationMessage);
+            }
+            catch (Exception)
+            {
+                jobSucceeded = false;
+                throw;
+            }
+            finally
+            {
+                await _jobHistoryService.UpdateJobHistory(GetType().Name, jobSucceeded);
+            }
+        }
```

Comment:
I think to pass the sonar cloud analysis you'll need to change the functions so the logic isnt the same in both functions, and is in a common function that they both call. Like:
```
[function]
NewRunHttpFunction()
{
  RunMethod()
}

[function]
OriginalRunFunction
{
  RunMethod()
}

async task RunMethod(){
    [old logic from OriginalRunFunction]
}
```